strings/names/lizard_verbs.txt

```diff
@@ -0,0 +1,80 @@
+Offends
+Strikes
+Orders
+Swallows
+Paralyzes
+Switches
+Tastes
+Pierces
+Tickles
+Places
+Pleases
+Toasts
+Poisons
+Transforms
+Possesses
+Tweaks
+Prepares
+Twists
+Promises
+Turns
+Protects
+Tosses
+Purchases
+Punches
+Understands
+Unlocks
+Unloads
+Raises
+Uses
+Recognises
+Upgrades
+Vacates
+Removes
+Wants
+Warms
+Washes
+Watches
+Selects
+Slaps
+Smells
+Wipes
+Wraps
+Spits
+Wrecks
+Gathers
+Keeps
+Grabs
+Grasps
+Kills
+Grips
+Kisses
+Greases
+Knocks
+Handles
+Hangs
+Lays
+Honours
+Leaves
+Hurts
+Helps
+Imitates
+Loads
+Loves
+Lowers
+Maintains
+Inspects
+Massages
+Melts
+Mocks
+Munches
+Murders
+Notices
+Judges
+Bends
+Owns
+Beats
```

Comment:
Just..... Just......

Reply:
Beats-The-Administrator

Reply:
At long last, I can have the anti-NT red-colored lizard with a trenchcoat and officer's hat.
Owns-The-Means